src/components/Home/NewsCategoriesGrid.tsx

```diff
@@ -0,0 +1,124 @@
+
+                // store the pattern data of the current batch of cards
+                let pattern: CardsPattern
+                // index of the current category
+                let index = 0
+                // number of categories remaining
+                let remaining = categories.length
+
+                while (index < categories.length) {
+                    // choose the dimension of the next batch of cards and get the corresponding pattern
+                    if (index === 0) {
+                        // When first bacth
+                        if (remaining <= 6) {
+                            pattern = newsCategoryPatterns.first[remaining]
+                        } else {
+                            pattern = newsCategoryPatterns.first[4]
+                        }
+                    } else {
+                        // When other batches. Here it is never possible that remaining === 1 or remaining === 2
+                        if (remaining % 5 === 1 || remaining % 5 === 3) {
+                            pattern = newsCategoryPatterns.other[3]
+                        } else if (remaining % 5 === 2 || remaining % 5 === 4) {
+                            pattern = newsCategoryPatterns.other[4]
+                        } else {
+                            pattern = newsCategoryPatterns.other[5]
+                        }
+                    }
+                    // create all the cards using the data in the pattern and append them to the correct column
+                    for (const [height, column] of pattern) {
+                        const title = categories[index].name
+                        tempCards[column].push(
+                            <CardWithGradient
+                                key={index}
+                                title={title}
+                                imageURL={categories[index].image}
+                                onClick={() =>
+                                    navigation.navigate("NewsList", {
+                                        categoryName: title,
+                                    })
+                                }
+                                closerToCorner={true}
+                                style={{ height: height }}
+                            />
+                        )
+                        index++
+                        remaining--
+                    }
+                }
+                setCards(tempCards)
+            })
+            .catch(error => console.log(error))
+    }, [])
+
+    return (
+        <View style={{ paddingBottom: 120 }}>
+            <CardWithGradient
+                // the news highlight card
+                title={"In Evidenza"}
+                imageURL={undefined} // TODO: update with correct highlight image
+                onClick={() =>
+                    navigation.navigate("NewsList", {
+                        categoryName: "In Evidenza",
```

Comment:
per qualche ragione la card si chiama "In Evidenza" ma la categoria nella pagina si chiama solo "Evidenza"
```suggestion
                        categoryName: "Evidenza",
```

Reply:
Ok, dovrei aver fatto.

Ho esteso gli stessi suggerimenti anche al file `NewsCategoriesGrid`